Userland/Libraries/LibSoftGPU/Device.h

```diff
@@ -66,8 +66,8 @@ struct RasterizerOptions {
     WindingOrder front_face { WindingOrder::CounterClockwise };
     bool cull_back { true };
     bool cull_front { false };
-    u8 texcoord_generation_enabled_coordinates { TexCoordGenerationCoordinate::None };
-    Array<TexCoordGenerationConfig, 4> texcoord_generation_config {};
+    Array<u8, NUM_SAMPLERS> texcoord_generation_enabled_coordinates { TexCoordGenerationCoordinate::None };
```

Comment:
I think this initializes only one element, and default initializes the other? The initial value can probably be dropped anyway

Reply:
Yes good catch

Reply:
I'm simply default initializing this now. It gets set by LibGL anyway if we enable texcoord generation.